R/gtrends.R

```diff
@@ -216,6 +216,7 @@ gtrends <- function(
 #' }
 plot.gtrends <- function(x, ...) {
   df <- x$interest_over_time
+  df$hits <- as.numeric(gsub('<','',df$hits))
```

Comment:
That is not a bad idea, but I think I;d be more defensive and first test if  `df$hits` is actually of type `character`, and if so do the `gsub()` and conversion.  Ideally we'd also catch an error on that...